rockcraft.yaml

```diff
@@ -0,0 +1,115 @@
+name: ceph
+base: ubuntu:22.04 # the base environment for this ROCK
+version: '0.1' # just for humans. Semantic versioning is recommended
+summary: Ubuntu based Ceph container image
+description: |
+    Rock for Containerised Ceph based on Ubuntu Ceph distribution. 
+license: MIT
+platforms:
+    amd64:
+
+package-repositories:
+  - type: apt
+    ppa: utkarshbhatthere/ceph-oci
```

Comment:
Cosmetic: maybe add a comment that this ppa should be replaced with ubuntu archives once patches land

Reply:
Ack